src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,63 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  onMovieAdd: (movie: Movie) => void;
+}
+
+const defaultImg = 'https://via.placeholder.com/360x270.png?text=no%20preview';
```

Comment:
usually such things are placed to some neighboring file, like `FindMovie.constants.ts`
```js
export const DEFAULT_IMG = 'https://via.placeholder.com/360x270.png?text=no%20preview';
```